docs/help/changes.md

```diff
@@ -32,4 +32,22 @@ App services are no longer sticky by default. Sticky sessions can be enabled in
 
     services
       web:
-        sticky: true
\ No newline at end of file
+        sticky: true
+
+### Timer Syntax
+
+Timers no longer follow the AWS [scheduled events](https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html) syntax where you must have a `?` in either day-of-week or day-of-month column. 
```

Comment:
This would also be a great candidate to add here:

https://github.com/convox/convox/blob/28d436744dbcd7b6e10db7d5c244aa8a5847cdfd/pkg/manifest/validate.go#L76